lib/tests/streamlit/elements/vega_charts_test.py

```diff
@@ -1324,7 +1324,7 @@ def test_chart_with_ordered_categorical_data(
 
         assert chart_spec["mark"] in [altair_type, {"type": altair_type}]
         assert chart_spec["encoding"]["x"]["type"] == "ordinal"
-        assert chart_spec["encoding"]["x"]["sort"] == ["c", "b", "a"]
+        assert chart_spec["encoding"]["x"]["sort"] is None
```

Comment:
Note sure if this test still makes sense now or if there's another way to check that the ordered categories are applied properly, given that we're disabling sorting now by setting `sort=None`.

Reply:
I just ran this manually and it seems like it's ordered b -> c -> a, i.e. in the order of the data. Not sure how before this even worked to get the right order from the Pandas dataframe, given that Altair should sort the categories alphabetically by default?

Reply:
Resolved this by not doing any special casing but allowing `sort=True` to get the old behavior with Altair's automatic sorting back. (The drawback of special casing is that then there's no way to disable this sorting...).